yepun/kuma/ingress-kuma.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/backend-protocol: HTTP
+    nginx.ingress.kubernetes.io/server-snippet: |
+      proxy_ssl_verify off;
+  name: it-kuma-dashboard
+  namespace: kuma
+spec:
+  rules:
+  - host: it-kuma01.cp.lsst.org
+    http:
+      paths:
+      - backend:
+          service:
+            name: my-uptime-kuma
+            port:
+              name:
+              number: 3001
+        path: /
+        pathType: Prefix
+  tls:
+  - hosts:
+    - it-kuma01.cp.lsst.org
+    secretName: it-kuma01-dashboard-ingress-tlsi
```

Comment:
Is the trailing `i` a typo?

Reply:
Correct, that is indeed a typo. I will update the ingress.